src/interfaces/shoppingCartInterface.ts

```diff
@@ -0,0 +1,5 @@
+import { Product } from 'src/interfaces/productInterface';
+
+export interface ShoppingCart{
+    productsInCart: Product[];
```

Comment:
"productsInCart" may be a little redundant since it's in a ShoppingCart object already. 
```suggestion
    products: Product[];
```